cli/schema/cypress.schema.json

```
@@ -278,6 +278,76 @@
           "type": "boolean",
           "default": false,
           "description": "Enables including elements within the shadow DOM when using querying commands (e.g. cy.get(), cy.find()). Can be set globally in cypress.json, per-suite or per-test in the test configuration object, or programmatically with Cypress.config()"
+        },
+        "clientPkiCertificates": {
+          "description": "Defines client PKI certificates to use when sending requests to the specified URLs",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "url": {
+                "description": "URL matching string",
```

It would be nice to supply more details here.
```suggestion
                "description": "Requests for URLs matching this minimatch pattern will use the supplied client certificate.",
```